blog/content/post/2024-04-11-aem_phase4.md

```diff
@@ -0,0 +1,257 @@
+few other, usually smaller changes were made:
+
+- Script for dumping DRTM TPM event log was enhanced to support event log header
+  as defined by TCG for TPM1.2 before the one specific to TXT. SKL puts TXT
+  header as part of vendor info in TCG header, this gives a good compromise
+  between two very similar formats - entries are identical, but headers are not.
+- Scriptlet that creates grub.cfg file now searches for SKL, in addition to
+  SINIT ACMs.
+- GRUB2 is now able to choose best `slaunch_module` from all passed ones. This
+  should allow users to put all ACMs and SKL in `/boot` directory, without
+  having to choose the proper one.
+- This release is published in a form of repository to (hopefully) make
+  installation process easier.
+
+List of pull requests created and merged as part of this phase:
+
+- [GRUB #17](https://github.com/TrenchBoot/grub/pull/17)
+- [GRUB #18](https://github.com/TrenchBoot/grub/pull/18)
+- [GRUB #19](https://github.com/TrenchBoot/grub/pull/19)
+- [Xen #9](https://github.com/TrenchBoot/xen/pull/9)
+- [Xen #10](https://github.com/TrenchBoot/xen/pull/10)
+- [Xen #12](https://github.com/TrenchBoot/xen/pull/12)
+- [Xen #13](https://github.com/TrenchBoot/xen/pull/13)
+- [Secure Kernel Loader #14](https://github.com/TrenchBoot/secure-kernel-loader/pull/14)
+- [Secure Kernel Loader #16](https://github.com/TrenchBoot/secure-kernel-loader/pull/16)
+- [Secure Kernel Loader #17](https://github.com/TrenchBoot/secure-kernel-loader/pull/17)
+- [Secure Kernel Loader #18](https://github.com/TrenchBoot/secure-kernel-loader/pull/18)
+- [Secure Kernel Loader #19](https://github.com/TrenchBoot/secure-kernel-loader/pull/19)
+- [Qubes Anti Evil Maid #9](https://github.com/TrenchBoot/qubes-antievilmaid/pull/9)
+- [Qubes Anti Evil Maid #10](https://github.com/TrenchBoot/qubes-antievilmaid/pull/10)
+- [Qubes Anti Evil Maid #11](https://github.com/TrenchBoot/qubes-antievilmaid/pull/11)
+
+## Installation
+
+To install, you have to first add a new repository and import a public part of
+a key that was used to sign RPM packages. To add a new repository, create in
+dom0 as root `/etc/yum.repos.d/aem.repo` with the following content:
+
+```text
+[aem]
+name = Anti Evil Maid based on TrenchBoot
+baseurl = https://dl.3mdeb.com/rpm/QubesOS/r4.2/current/dom0/fc37
+gpgcheck = 1
+gpgkey = https://dl.3mdeb.com/rpm/QubesOS/r4.2/current/dom0/fc37/RPM-GPG-KEY-tb-aem
+enabled = 1
+```
+
+![Content of aem.repo](/img/qubes_aem_repo.png)
+
+To download and import a signing key:
+
+```bash
+qvm-run --pass-io sys-net 'curl -L https://dl.3mdeb.com/rpm/QubesOS/r4.2/current/dom0/fc37/RPM-GPG-KEY-tb-aem' > RPM-GPG-KEY-tb-aem
+sudo rpm --import RPM-GPG-KEY-tb-aem
+```
+
+![Importing signing key](/img/qubes_aem_import_key.png)
+
+Now it should be possible to download and install packages from our repository.
+As some of the packages are also available in standard QubesOS repositories,
+potentially in never versions, those must be temporarily disabled during
```

Comment:
```suggestion
potentially in newer versions, those must be temporarily disabled during
```

Reply:
:man_facepalming:

Reply:
https://github.com/3mdeb/news-and-ideas/pull/592/commits/36bcaf6c3b67070087da7359d81493343e171ad8